packages/js/package-lock.json

```diff
@@ -0,0 +1,2128 @@
+{
```

Comment:
Why do we even have this lock file since we are using pnpm?

Reply:
Removed.